docs/deployment/manage-projects.md

```diff
@@ -2,31 +2,28 @@
 slug: manage-projects
 title: Manage projects
 hide_title: true
-description: View projects, detailed logs and information for any scan.
+description: View projects, detailed logs, and information for any scan.
 tags:
-    - Deployment
-    - Semgrep AppSec Platform
+ - Deployment
+ - Semgrep AppSec Platform
 ---
 
 # Manage projects
 
-View, sort, and tag your projects through the **Projects** page. A **project** is any repository that you have scanned with Semgrep AppSec Platform through CI or Semgrep Managed Scans, and includes that repository's findings and metadata.
+View, sort, and tag your projects through the **Projects** page. Refer to this page to **manage** and **troubleshoot** thousands of repositories by identifying scan issues or scans with a high number of findings.
 
-Refer to this page to **manage** and **troubleshoot** thousands of repositories by identifying scan issues or scans with a high finding count.
+:::note What is a project?
+A **project** is any repository that you have scanned with Semgrep AppSec Platform through CI or Semgrep Managed Scans. It includes that repository's findings and metadata.
```

Comment:
You may want to talk to @chmccreery about the fact that this will be a bit different going forward :)